scripts/bundleSize.js

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env node
+
+/* eslint-disable no-console */
```

Comment:
You have a `// eslint-disable-line no-console` down there, can this be removed? IMO single line disables are nicer :)